packages/edit-widgets/src/store/index.js

```diff
@@ -35,10 +35,7 @@ const storeConfig = {
  * @type {Object}
  */
 export const store = createReduxStore( STORE_NAME, storeConfig );
-
-// Once we build a more generic persistence plugin that works across types of stores
-// we'd be able to replace this with a register call.
-registerStore( STORE_NAME, storeConfig );
```

Comment:
Just for my own clarity: Is there something in this PR that lets us remove this that I'm not understanding? Or is it a "while we're here" change?

Reply:
It's a 'while we're here' change. It should have been done previously when the edit-widgets store stopped using the persistence plugin (in https://github.com/WordPress/gutenberg/pull/33774).

Thanks for reviewing.

Reply:
Ahhh gotcha.